src/main/java/com/nike/cerberus/command/core/GenerateCertsCommand.java

```diff
@@ -0,0 +1,177 @@
+/*
+ * Copyright (c) 2017 Nike, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nike.cerberus.command.core;
+
+import com.beust.jcommander.Parameter;
+import com.beust.jcommander.Parameters;
+import com.nike.cerberus.command.Command;
+import com.nike.cerberus.operation.Operation;
+import com.nike.cerberus.operation.core.GenerateCertsOperation;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static com.nike.cerberus.command.core.GenerateCertsCommand.COMMAND_DESCRIPTION;
+import static com.nike.cerberus.command.core.GenerateCertsCommand.COMMAND_NAME;
+
+@Parameters(
+        commandNames = COMMAND_NAME,
+        commandDescription = COMMAND_DESCRIPTION
+)
+public class GenerateCertsCommand implements Command {
+
+    public static final String COMMAND_NAME = "generate-certs";
+    public static final String COMMAND_DESCRIPTION = "Generates the TLS certificates needed to enable https " +
+            "through out the system, using an ACME provider such as LetsEncrypt";
+
+    public static final String BASE_DOMAIN_LONG_ARG = "--base-domain";
+    public static final String BASE_DOMAIN_SHORT_ARG = "-d";
+
+    public static final String HOSTED_ZONE_ID_LONG_ARG = "--hosted-zone-id";
+    public static final String HOSTED_ZONE_ID_SHORT_ARG = "-h";
+
+    public static final String ADDITIONAL_SUBJECT_ALT_NAME_LONG_ARG = "--additional-subject-alternative-name";
+    public static final String ADDITIONAL_SUBJECT_ALT_NAME_SHORT_ARG = "-a";
+
+    public static final String ENABLE_LE_CERTFIX_LONG_ARG = "--enable-letsecrypt-certfix";
+    public static final String ENABLE_LE_CERTFIX_SHORT_ARG = "-elec";
```

Comment:
I'd be okay with dropping the short arg names.  This command isn't ever going to be ran so frequently that the short names are all that helpful.  Also, it will be confusing if we ever want to use a short name as a global parameter later.

I'm okay with leaving them too but my first preference is to drop them.

Reply:
I dropped them